src/software/gui/visualizer/widgets/ai_control.cpp

```diff
@@ -100,23 +101,22 @@ void setupDefendingSideComboBox(Ui::AutoGeneratedMainWidget *widget,
     widget->defending_side_combo_box->insertItem(2, "West");
 
     auto on_defending_side_changed = [config](const QString &text) {
-        auto refbox_config =
-            config->getMutableAIControlConfig()->getMutableRefboxConfig();
+        auto sensor_fusion_config = config->getMutableSensorFusionConfig();
         if (text == "Use Refbox")
         {
-            refbox_config->mutableOverrideRefboxDefendingSide()->setValue(false);
+            sensor_fusion_config->mutableOverrideRefboxDefendingSide()->setValue(false);
         }
         else if (text == "East")
         {
-            refbox_config->mutableOverrideRefboxDefendingSide()->setValue(true);
+            sensor_fusion_config->mutableOverrideRefboxDefendingSide()->setValue(true);
             // TODO: Confirm how East and West map to positive and negative sides
-            refbox_config->mutableDefendingPositiveSide()->setValue(false);
+            sensor_fusion_config->mutableDefendingPositiveSide()->setValue(false);
         }
         else if (text == "West")
         {
```

Comment:
can we fix this in this PR? Shouldn't be too hard...

Reply:
Should be easy but I'm going to call that out of scope and fix it with my other Visualizer changes this week.